### PR TITLE
Session now retrieved from task.requests, respecting domain_delays an…

### PR DESCRIPTION
**Pull request by qvazzler**
…d other task config items. Also added default domain delay of 4 seconds due to increased spam filters by SCC

Planning to do this for most search plugins, at least to use the task.requests session.
This is a needed fix in order to use SCC currently.

### CODE REVIEW

File: flexget/plugins/search_sceneaccess.py [hunk 1]
```diff
@@ -234,6 +235,14 @@ def search(self, task, entry, config=None):
             Search for entries on SceneAccess
         """
 
+        try:
+            session = task.requests
+        except:
```
Comment (cvium) [inline flexget/plugins/search_sceneaccess.py]: catching everything like this is ill-advised. I also don't see it ever throwing an exception. You're accessing an object member that is set on object creation.

File: flexget/plugins/search_sceneaccess.py [hunk 2]
```diff
@@ -10,11 +10,11 @@
 from flexget.event import event
 from flexget.utils.soup import get_soup
 from flexget.utils.search import torrent_availability, normalize_unicode, clean_title
-from flexget.utils.requests import Session
+from flexget.utils.requests import Session, TimedLimiter
 
 log = logging.getLogger('search_sceneaccess')
 
-session = Session()
+session = None
```
Comment (cvium) [inline flexget/plugins/search_sceneaccess.py]: Is it necessary to declare the variable in global scope? Isn't it only used in `search`?

File: flexget/plugins/search_sceneaccess.py [hunk 3]
```diff
@@ -10,11 +10,11 @@
 from flexget.event import event
 from flexget.utils.soup import get_soup
 from flexget.utils.search import torrent_availability, normalize_unicode, clean_title
-from flexget.utils.requests import Session
+from flexget.utils.requests import Session, TimedLimiter
```
Comment (cvium) [inline flexget/plugins/search_sceneaccess.py]: Is the `Session` import used anymore?

File: flexget/plugins/search_sceneaccess.py (hunk 2 above)
Comment (qvazzler) [inline flexget/plugins/search_sceneaccess.py]: For now, just want to replicate previous code where possible.

File: flexget/plugins/search_sceneaccess.py [hunk 4]
```diff
@@ -234,6 +232,11 @@ def search(self, task, entry, config=None):
             Search for entries on SceneAccess
         """
 
+        session = task.requests
+
+        if not 'sceneaccess.eu' in session.domain_limiters:
```
Comment (cvium) [inline flexget/plugins/search_sceneaccess.py]: pep standard prefers `if string not in bla bla` and I'm a firm believer in adhering to standards *hint hint*

Comment (qvazzler) [inline flexget/plugins/search_sceneaccess.py]: Very well..!

**Comment by cvium** [comment]
Is it ready then?

**Comment by qvazzler** [comment]
Sure feels like it! @gazpachoking

**Pull request merged**